src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+@import "variables";
+
+.card {
+  width: 167px;
+  height: 360px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: block;
+
+  &__pic {
+   display: block;
+   margin: 0 auto;
+   height: 134px;
+   width: 160px;
+  }
+
+  &__name {
+    color: $text-color;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 140%;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    color: $text-color;
+    font-size: 10px;
+    margin-top: 3px;
+    text-align: center;
+  }
+
+  &__reviews-wrap{
+    height: 16px;
+  }
+
+  &__reviews-wrap,
+  &__price-wrap {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    color: $secondary-color;
+    font-size: 12px;
+    line-height: 150%;
+    margin-top: 24px;
+    margin-bottom: 16px;
+    display: inline-block;
+
+    &_sum {
+      color: $text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      margin-top: 24px;
+      margin-bottom: 16px;
+      display: inline-block;
+    }
+  }
+
+  &__buy {
+    padding: 11px;
+    background-color: $blue-color;
+    border: 1px solid $blue-color;
+    border-radius: 5px;
+    display: block;
+
+    color: #fff;
+    text-decoration: none;
+    text-transform: uppercase;
+    text-align: center;
+    line-height: 16px;
+    font-weight: 700;
+    font-size: 14px;
+  }
+
+  &__buy:hover {
+    background-color: #fff;
+    color: $blue-color;
+  }
+}
+
+.card:hover {
+  transform: scale(1.2);
+}
```

Comment:
you need to make changes on hover smooth, add transition, but don't add it on hover because it will not work right way